roles/uperf-scale/tasks/start_server.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- block:
```

Comment:
You could have remove this "block" since we do not support VM kind in scale mode. But for the sake of reviewing convenience, leave the "block" alone reduces the diff greatly.